.github/workflows/groovy-joint-workflow.yml

```diff
@@ -78,14 +78,9 @@ jobs:
           echo "" >> $GITHUB_OUTPUT
           echo "buildCache {" >> $GITHUB_OUTPUT 
           echo "    local { enabled = System.getenv('CI') != 'true' }" >> $GITHUB_OUTPUT 
-          echo "    remote(HttpBuildCache) {" >> $GITHUB_OUTPUT 
+          echo "    remote(gradleEnterprise.buildCache) {" >> $GITHUB_OUTPUT 
           echo "        push = System.getenv('CI') == 'true'" >> $GITHUB_OUTPUT
```

Comment:
Does the push flag need to be enabled only when GRADLE_ENTERPRISE_ACCESS_KEY is present? 
Like in [here](https://github.com/grails/grails-core/pull/13387/files#diff-7f825392aa37acd1cee0c2e7b9bb7366ad6eac64f3e6cdd816e156bcb69d30deR23)